Revamp README.md to contain more beginner-friendly informations

cfillion opened the pull request:
I tried to improve the documentation of this repository a little bit:
- Added basic installation/contributing instructions in README.md (similar to what's in JSFX's readme)
- Moved part of it in a special ABOUT.md file for ReaPack's [about dialog](https://i.imgur.com/4ywfTMf.png):
  - Installation instructions are not useful if the repository is already installed... 😛 
  - Since the ReaTeam member list is already available [here](https://github.com/orgs/ReaTeam/people), it's probably more useful in ABOUT.md than the README
- Removed the "code taken from" section. I think it would be better to write that information in the scripts themselve, if required, rather than in the readme.

Preview: [README.md](https://github.com/ReaTeam/ReaScripts/blob/new-readme/README.md) & [ABOUT.md](https://github.com/ReaTeam/ReaScripts/blob/new-readme/ABOUT.md)

ping @MichaelPilyavskiy as you wrote the original readme: do you have comments on this before merging?
(Also feel free to make other changes to these files in the [`new-readme`](https://github.com/ReaTeam/ReaScripts/tree/new-readme) branch if you want)

cfillion committed:
revamp README.md to contain more infos

cfillion committed:
move part of README.md in ABOUT.md

MichaelPilyavskiy commented:
Everything looks better and more clear, so I think it needs to merge this.
Thanks!

X-Raym commented:
Because not everyoone knows git files, and because ABOUT.md contains precious informations about ReaTeam and its member (list, donation link etc), I think it would be cool to add a link to ABOUT.md right from the README.md.

cfillion commented:
Indeed. I also think it should list every contributor, not only those that joined ReaTeam.

Modified both files in the `new-readme` branch:
- Added link to ABOUT.md at the top of README.md
- Reorganized ABOUT.md to contain only a single list with
  - link to forum profile
  - website link if any
  - donation link if any

![Preview](https://i.imgur.com/xGxUjSy.png)
